examples/closure-app/closure-common/src/ClosureDimensionEndpoint.cpp

```diff
@@ -67,12 +67,32 @@ CHIP_ERROR ClosureDimensionEndpoint::Init()
 
 void ClosureDimensionEndpoint::OnStopMotionActionComplete()
```

Comment:
There are a lot of assumptions here that aren't defined in the spec - these need to be discussed.